includes/definitions/discovery/dell-net.yaml

```diff
@@ -0,0 +1,80 @@
+mib: DELL-NETWORKING-CHASSIS-MIB:DELL-NETWORKING-IF-EXTENSION-MIB
+modules:
+    processors:
+        data:
+            -
+                snmp_flags: ['-ObsQ']
+                oid: dellNetCpuUtilTable
+                value: dellNetCpuUtil5Sec
+                num_oid: '.1.3.6.1.4.1.6027.3.26.1.4.4.1.1.{{ $index }}'
+                descr: '5s Utilization'
+                index: 'a_dellNetCpuUtil5Sec.{{ $index }}'
+            -
+                snmp_flags: ['-ObsQ']
+                oid: dellNetCpuUtilTable
+                value: dellNetCpuUtil1Min
+                num_oid: '.1.3.6.1.4.1.6027.3.26.1.4.4.1.4.{{ $index }}'
+                descr: '1m Utilization'
+                index: 'b_dellNetCpuUtil1Min.{{ $index }}'
+            -
+                snmp_flags: ['-ObsQ']
+                oid: dellNetCpuUtilTable
+                value: dellNetCpuUtil5Min
+                num_oid: '.1.3.6.1.4.1.6027.3.26.1.4.4.1.5.{{ $index }}'
+                descr: '5m Utilization'
+                index: 'c_dellNetCpuUtil5Min.{{ $index }}'
+            -
+                snmp_flags: ['-ObsQ']
+                oid: dellNetCpuUtilTable
+                value: dellNetCpuUtilMemUsage
+                num_oid: '.1.3.6.1.4.1.6027.3.26.1.4.4.1.6.{{ $index }}'
+                descr: 'Memory'
+                index: 'd_dellNetCpuUtilMemUsage.{{ $index }}'
```

Comment:
This one looks like MemUsage and should not be included in the Processor category.

Reply:
commit 65a41331c782b5a12db3ec5831e7310013a88690 moves this to a proper mempool